### DISCUSSION TITLE
Mysql Connections Count is not accurate

### DISCUSSION THREAD

**Discussion by Saifallak** [spatie/laravel-health#198] [Q&A] [answered]
as you can see here:
https://github.com/spatie/laravel-health/blob/1dee638806b6b2d8971363bf1e26cb17835e2dcb/src/Support/DbConnectionInfo.php#L15

we're using `show status where variable_name = "threads_connected"` which shows overall server processes not my user-only
this is wrong when running several apps on the same server and setting `max_user_connections` to like 60 per app.
it should warn me when my user exceeds the limit not the overall server.

so it should be changed to `SELECT COUNT(*) FROM information_schema.PROCESSLIST;`

this won't be breaking change for others, as if you're the only user on the server or using root mysql, this will return same data,
but if user is shared with other apps or other users (ex: Shared Hosting) it will return his processes only,
so he can be notified when his app is doing something wrong.

tell me what you think and i can open PR.

**Comment by Saifallak** [comment]
@freekmurze do you agree for this? I can make a pr

**Reply by freekmurze** [reply, marked as answer]
Open up a PR and I'll have a look  👍

**Reply by Saifallak** [reply]
okay i will tomorrow thanks,

**Reply by Saifallak** [reply]
https://github.com/spatie/laravel-health/pull/202
